config/locales/cy.yml

```diff
@@ -153,6 +153,12 @@ toplevel:
       lastName: Cyfenw
       organisation: Mudiad
       emailAddress: Cyfeiriad e-bost
+    errors:
+      firstName: Please provide your first name
+      lastName: Please provide your last name
+      emailMissing: Please provide your email address
+      emailInvalid: Please provide a valid email address
+      location: Please choose a country
```

Comment:
We have most of this already:

`Please provide the '%s' field` = `Darparwch y maes '%s'`
`Please provide a valid email address` = `Rhowch gyfeiriad e-bost dilys`

You could probably translate this yourself by just adding in the field names?